docs/model_configuration.md

```diff
@@ -454,16 +454,45 @@ value. For the above example, an inference request must provide inputs
 with the following shapes.
 
 ```
-  "input0": [ x, -1]
-  "input1": [ 1 ]
-  "output0": [ x, -1]
+  "input0": [ x, 1, 3]
+  "input1": [ 3 ]
+  "output0": [ x, 1, 3]
 ```
 
 Where *x* is the batch size of the request. Triton requires the shape
 tensors to be marked as shape tensors in the model when using
-batching. Note that "input1" has shape *[ 1 ]* and not *[ 2 ]*. Triton
-will prepend the shape value *x* at "input1" before issuing the
-request to model.
+batching. Note that "input1" has shape *[ 3 ]* and not *[ x, 2 ]*. Triton
+will accumulate all the shape values together for "input1" in batch
+dimension before issuing the request to model.
+
+For example, assume the client sends following three requests to Triton
+with following inputs:
+
+```
+Request1:
+input0: [[[1,2,3]]] <== shape of this tensor [1,1,3]
+input1: [1,4,6] <== shape of this tensor [3]
+
+Request2:
+input0: [[[4,5,6]], [[7,8,9]]] <== shape of this tensor [2,1,3]
+input1: [2,4,6] <== shape of this tensor [3]
+
+Request3:
+input0: [[[10,11,12]]] <== shape of this tensor [1,1,3]
+input1: [1,4,6] <== shape of this tensor [3]
+```
+
+Assuming these requests get batched together would be delivered to the
+model as:
+
+
+```
+Batched Requests to model:
+input0: [[[1,2,3]], [[4,5,6]], [[7,8,9]], [[10,11,12]]] <== shape of this tensor [4,1,3]
+input1: [4, 4, 6] <== shape of this tensor [3]
```

Comment:
Should explain more about shape tensor is only for TensorRT and in what case (example?) it will be needed

Reply:
Sure!